### PR TITLE
feat: add multiple child project support to dot net core transformer as well

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: assets/built-in/transformers/dockerfilegenerator/dotnetcore/templates/Dockerfile [hunk 1]
```diff
@@ -12,32 +12,50 @@
...
-FROM mcr.microsoft.com/dotnet/sdk:{{ .DotNetVersion }} AS builder
+{{ if .IncludeBuildStage }}
+
+# Build Stage
+FROM mcr.microsoft.com/dotnet/sdk:{{ .BuildStageImageTag }} AS {{ .BuildContainerName }}
+
+{{- if .IsNodeJSProject }}
+RUN apt-get update -y && \
+    apt-get install -y xz-utils && \
+    wget https://nodejs.org/dist/v16.16.0/node-v16.16.0-linux-x64.tar.xz && \
```
Comment (ashokponkumar) [inline assets/built-in/transformers/dockerfilegenerator/dotnetcore/templates/Dockerfile:23]: Shouldn't this be dynamic?

File: common/utils.go [hunk 2]
```diff
@@ -564,16 +576,6 @@ func NormalizeForEnvironmentVariableName(envName string) string {
 	return newName
 }
 
-// IsStringPresent checks if a value is present in a slice
-func IsStringPresent(list []string, value string) bool {
```
Comment (ashokponkumar) [inline common/utils.go:568]: IsStringPresent does a case insensitive comparison, that's why we are not able to use generic for strings and ints.

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

**Comment by HarikrishnanBalagopal** [comment]
Fixes #835

File: assets/built-in/transformers/dockerfilegenerator/dotnetcore/templates/Dockerfile (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply assets/built-in/transformers/dockerfilegenerator/dotnetcore/templates/Dockerfile:23]: After this PR is merged https://github.com/konveyor/move2kube/pull/834 we can use the common logic.

File: common/utils.go (hunk 2 above)
Comment (HarikrishnanBalagopal) [reply common/utils.go:568]: It might be ok to sacrifice the flexibility there to gain clarity in code and usage.

**Comment by HarikrishnanBalagopal** [comment]
Fixes #839

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**